### PR TITLE
KOGITO-2613 add OUIA props in common/Atoms

### PR DESCRIPTION
**Pull request by jstastny-cz**
This is first of many PRs that will incrementally provide OUIA component attributes.
This is for Atom components from common package.

Changes present:
* OuiaUtils - not anymore dependent on ouiaContext and withOuiaContext HOC - limited changes in snapshots.
* Components - added OUIAProps interface with ouiaId, ouiaSafe props, using componentOuiaProps helper to actually set those on top-level element of the component
* Snapshots
  * in tests for the actual components the ouia attributes are present in snapshots - I take it as benefit - we need to be careful about changes in such attributes and snapshot review will identify those.
  * in tests for higher-level components -> using mocking where applicable.

### CODE REVIEW

**Review by yzhao583 — APPROVED** [review]
Looks good to me thanks.

**Review by pefernan — APPROVED** [review]
+1

**Comment by jstastny-cz** [comment]
jenkins retest this

File: ui-packages/packages/management-console/src/components/Templates/ProcessDetailsPage/tests/ProcessDetailsPage.test.tsx [hunk 1]
```diff
@@ -62,6 +62,8 @@ jest.mock('@kogito-apps/common', () => ({
   }
 }));
 
+jest.mock('../../../Organisms/ProcessDetails/ProcessDetails');
```
Comment (jstastny-cz) [inline ui-packages/packages/management-console/src/components/Templates/ProcessDetailsPage/tests/ProcessDetailsPage.test.tsx]: duplicated mock - need to remove

**Comment by Sara4994** [comment]
looks good :+1:

**Review by Sara4994 — APPROVED** [review]

**Review by AjayJagan — APPROVED** [review]
:+1:

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**